tests/e2e-upgrade/upgrade-test/01-upgrade-operator.yaml

```diff
@@ -1,6 +1,5 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-  - command: kubectl apply -f ../../_build/manifests/01-opentelemetry-operator.yaml
-  - command: kubectl rollout status -w deployment/opentelemetry-operator-controller-manager -n opentelemetry-operator-system
-  - command: sleep 60s
+  - script: cd ../../../ && make deploy VERSION=e2e
+  - command: sleep 10
```

Comment:
why there is this sleep statement?

Reply:
Same reason as https://github.com/open-telemetry/opentelemetry-operator/pull/1254#discussion_r1037471952